listenbrainz_spark/recommendations/create_dataframes.py

```diff
@@ -115,25 +145,20 @@ def get_listens_for_training_model_window(to_date, from_date, metadata, dest_pat
     except FileNotFetchedException as err:
         current_app.logger.error(str(err), exc_info=True)
         sys.exit(-1)
-    return utils.get_listens_without_artist_and_recording_mbids(training_df)
+
+    partial_listens_df = utils.get_listens_without_artist_and_recording_mbids(training_df)
```

Comment:
Why this extra assignment?

Reply:
oops. I am closing it for now. Will update in a separate doc PR where other nitpicks needs to be addressed.